### PR TITLE
`interval` shouldn’t run `tick` after stop

### LINKED ISSUE #236: `interval`: `stop` does not clear timeout
Bug reproduced here https://share.effector.dev/8InYtWNj
Last interval should be cleared so `time` should be ~5000

https://github.com/effector/patronum/pull/225 fixed issue with `tick` but does not actually clear the timeout

### PR DESCRIPTION
**Pull request by ainursharaev**
Closes #236 

The solution is provided by @AlexandrHoroshih

### CODE REVIEW

**Review by sergeysova — APPROVED** [review]

File: src/interval/index.ts [hunk 1]
```diff
@@ -29,40 +19,48 @@ export function interval<S extends unknown, F extends unknown>({
...
+  sample({
     clock: start,
     source: $timeout,
-    filter: $notRunning,
     target: timeoutFx,
   });
 
   if (leading) {
-    guard({
+    sample({
```
Comment (AlexandrHoroshih) [inline src/interval/index.ts:63]: This change means that minimal required version of `effector` should be up to `22.2.0` 🤔 
Now is `22.1.2`, which does not have sample with filter yet - looks like this is breaking change and will lead to major update

I suggest to do it as a separate PR and leave guards in place in this one

What you think, @sergeysova ?

**Review by AlexandrHoroshih — COMMENTED** [review]
Everything else looks good 👍

File: src/interval/index.ts (hunk 1 above)
Comment (ainursharaev) [reply src/interval/index.ts]: I've put everything back the way it was before

**Review by AlexandrHoroshih — APPROVED** [review]

**Comment by sergeysova** [comment]
@ainursharaev Thank you!
